src/mongo/db/storage/wiredtiger/encryption_keydb.h

```diff
@@ -46,22 +47,59 @@ namespace mongo {
 class EncryptionKeyDB
 {
 public:
-    EncryptionKeyDB(const std::string& path, const bool rotation = false)
-        : EncryptionKeyDB(false, path, rotation) {}
-    EncryptionKeyDB(const bool just_created, const std::string& path)
-        : EncryptionKeyDB(just_created, path, false) {}
     ~EncryptionKeyDB();
 
-    // tries to read master key from specified file
-    // then opens WT connection
-    // throws exceptions if something goes wrong
-    void init();
-
-    // during rotation copies data from provided instance
-    void clone(EncryptionKeyDB *old);
-
-    // write master key to the Vault (during rotation)
-    void store_masterkey();
+    /// @brief Creates a (possibly encrypted at rest) new encryption key database.
```

Comment:
In fact encryption key db is always encrypted at rest.
So "possible encrypted at rest" is misleading here